src/js/actions.js

```diff
@@ -124,11 +123,7 @@ function submitToUrl(body, submitUrl, trackingPrefix) {
 
     req.setRequestHeader('X-Key-Inflection', 'camel');
     req.setRequestHeader('Content-Type', 'application/json');
-
-    const userToken = _.get('sessionStorage.userToken', window);
-    if (userToken) {
-      req.setRequestHeader('Authorization', `Token token=${userToken}`);
-    }
+    req.withCredentials = true;
```

Comment:
Just a note that this works on IE10 and later. I see markup on the VA site that seems to be trying to support IE9 and below but as far as I can tell it can't possibly work there. Not sure if there is an official browser support goal but IE11 is fine with me.